library/src/scala/quoted/QuoteContext.scala

```diff
@@ -82,6 +82,8 @@ trait QuoteContext { self: internal.QuoteUnpickler & internal.QuoteMatching =>
    *
    *  +- Tree -+- PackageClause
    *           +- Import
+   *           +- Export
```

Comment:
perhaps can do ImportOrExport supertype, but it seems most of the class hierarchy is flattened

Reply:
WDYT @nicolasstucki, should we expose the ImportOrExport supertype in tasty reflect or do we avoid exposing supertypes in the reflection api?

Reply:
It looks like it would complicate things